### ISSUE TITLE
map.getCenter() does not work with polarstereograhic maps

### ISSUE THREAD

**Issue by noiv** [Leaflet/Leaflet#2133]
Especially if one of the poles is within the bounds results are outside of the map. Currently I use this code to circumvent:

``` javascript
getCenter: function(map){

  var prj = map.options.crs.projection,
      sw  = map.getBounds()._southWest,
      ne  = map.getBounds()._northEast,
      psw = prj.project(sw),
      pne = prj.project(ne),
      x   = psw.x + (pne.x - psw.x)/2,
      y   = psw.y + (pne.y - psw.y)/2;
          
  return prj.unproject({x:x, y:y});

},
```
Works at least with EPSG:3413. L.ImageOverlay._reset() and ._animateZoom() have  same problem.

**Comment by mourner** [comment]
I think it might have been fixed in 1.0.0-beta.1, can you verify?

**Comment by perliedman** [comment]
Like @mourner says, this looks like it has been fixed in current master, although I can't verify since I don't have a polarstereograhic map to test with.

Closing for now, but feel free to add more info if this is still a problem.
